### DISCUSSION TITLE
ЦДБГ PD

### DISCUSSION THREAD

**Discussion by Asdasdasdasdsaddczx2** [octothorp-team/public-dobrograd#811] [Предложения]
Всех бесит что Департамент на ЦДБГ пустует, верно? Я предлогаю уже наконец кому то сделать дубликат ПД и сделать его серверными пропами, что бы он был всегда и не пустовал, почему НДБГ кто то занялся и сделал это а на ЦДБГ пофиг, давайте уже сделаем нормальным ПД на ЦДБГ.

**Comment by Kilo-7** [comment]
@Asdasdasdasdsaddczx2, сделали.
